### PR TITLE
Edits and new entries to style guide

### PR DESCRIPTION
**Pull request by sshukor**
British national (overseas)
leave to enter
leave to stay
permission to enter
permission to stay

### CODE REVIEW

**Comment by tomyems** [comment]
Content approved. Remind me next time you want to do it and I'll make your life slightly easier (by about 2 clicks!)

**Comment by sshukor** [comment]
Thanks Tom! Have a great weekend.

Steven Shukor
Senior Content Designer
Migration and Borders Technology Portfolio (MBTP)
Digital, Data and Technology (DDaT), Corporate Enablers

Home Office

Metro Point, 49 Sydenham Road, Croydon, CR0 2EU


________________________________

File: apps/docs/src/common/pages/content-style-guide.tsx [hunk 1]
```diff
@@ -42,6 +42,8 @@ const Page: FC<PageProps> = ({ location }) => (
     <p>Capitalised.</p>
     <h3 id="border control">border control</h3>
     <p>Use ‘border control’ rather than ‘passport control’ or ‘primary control point’.</p>
+    <h3 id="british national (overseas)">British national (overseas)</h3>
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/content-style-guide.tsx:45]: We should check that it's okay to have parentheses here...

**Review by daniel-ac-martin — APPROVED** [review]

File: apps/docs/src/common/pages/content-style-guide.tsx (hunk 1 above)
Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/content-style-guide.tsx:45]: Seems fine.

**Pull request merged**